src/Plugin/Purge/Purger/AkamaiPurger.php

```diff
@@ -93,4 +85,11 @@ public function invalidate(array $invalidations) {
     $this->client->purgeUrls($urls_to_clear);
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function hasRuntimeMeasurement() {
+    return TRUE;
+  }
+
```

Comment:
No, Purge takes care of all of this for you actually. Just by returning TRUE, its saying "yes, do this runtime tracking please"